templates/tapestry.js

```diff
@@ -37,6 +37,7 @@ function tapestryTool(config){
         API_DELETE_METHOD = 'DELETE',
         USER_NODE_PROGRESS_URL = config.apiUrl + "/users/progress",
         USER_NODE_UNLOCKED_URL = config.apiUrl + "/users/unlocked",
+        USER_NODE_SKIP_URL = config.apiUrl + "/users/skip",
```

Comment:
This should probably be named `skipped` to be consistent (everything else is a noun, not a verb)